app/core/routes.py

```
@@ -8,6 +8,12 @@
 
 main_bp = Blueprint('main', __name__)
 
+
+def _get_recent_topics(limit=5):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_get_recent_topics`

(ANN202)